### ISSUE TITLE
Add thread_ts as an optional parameter to use threads as part of the cypress cloud slack integration

### ISSUE THREAD

**Issue by a8trejo** [cypress-io/github-action#1150]
Hello cypress GHA team!

Asking for an enhancement proposal to the Cypress Github Action,, a great addition as an **optional** parameter for users of Cypress Cloud who use the slack integration would be `thread_ts`.

**Context**: In my company we have a slack channel used for deployment and test notifications (all environments), in order to keep the channel readable and as clean as possible, we would prefer to have the deployment notification as the main slack message, and on the thread include the Cypress slack report.

We're already using Slack's API to know the `thread_ts` which we would use for the cypress slack alert.

**Comment by MikeMcC399** [comment]
Hello @a8trejo 

Thank you for describing your use of Cypress GitHub Action with Cypress Cloud and the Slack integration.

Have you considered if you could use the [tag option](https://github.com/cypress-io/github-action/blob/master/README.md#tag-recordings) as this is already available in the [Cypress Cloud Slack integration](https://docs.cypress.io/guides/cloud/integrations/slack#Filter-notifications-by-tag)?

I'm not sure if this would meet your needs, but since this feature is already implemented it would be quick to put it into operation.

**Comment by a8trejo** [comment]
@MikeMcC399 yeah we're already using that, but you can use tags only to filter slack notifications, I **want** these notifications, just in a slack thread instead

**Comment by jennifer-shehane** [comment]
@a8trejo I forwarded this feedback to our Cloud team.

**Comment by a8trejo** [comment]
thank you very much @jennifer-shehane !
